src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -249,99 +237,53 @@ public Builder setCredentialsProvider(CredentialsProvider credentialsProvider) {
       return this;
     }
 
-    /**
-     * The previously set CredentialsProvider.
-     */
+    /** The previously set CredentialsProvider. */
     public CredentialsProvider getCredentialsProvider() {
       return credentialsProvider;
     }
 
-    /**
-     * Sets the address used to reach the service.
-     */
+    /** Sets the address used to reach the service. */
     public Builder setServiceAddress(String serviceAddress) {
       this.serviceAddress = serviceAddress;
       return this;
     }
 
-    /**
-     * The address used to reach the service.
-     */
+    /** The address used to reach the service. */
     public String getServiceAddress() {
       return serviceAddress;
     }
 
-    /**
-     * Sets the port used to reach the service.
-     */
+    /** Sets the port used to reach the service. */
     public Builder setPort(int port) {
       this.port = port;
       return this;
     }
 
-    /**
-     * The port used to reach the service.
-     */
+    /** The port used to reach the service. */
     public int getPort() {
       return port;
     }
 
-    /**
-     * Sets the generator name and version for the GRPC custom header.
-     */
-    public Builder setGeneratorHeader(String name, String version) {
-      this.serviceGeneratorName = name;
-      this.serviceGeneratorVersion = version;
+    /** Sets the generator name and version for the GRPC custom header. */
+    public Builder setClientHeader(String name, String version) {
```

Comment:
Should there be a corresponding `toolkit` change to pass these parameters from the `gccl` hand-written layer?